contracts/ERC721M.sol

```diff
@@ -65,6 +65,9 @@ contract ERC721M is IERC721M, ERC721AQueryable, Ownable, ReentrancyGuard {
     // Address of ERC-20 token used to pay for minting. If 0 address, use native currency.
     address private _mintCurrency;
 
+    address public constant MINT_FEE_RECEIVER =
+        address(0x0B98151bEdeE73f9Ba5F2C7b72dEa02D38Ce49Fc);
```

Comment:
we can get some gas savings by making this immutable since we don't have any update logic for it.

```
    address public immutable MINT_FEE_RECEIVER;

    constructor() {
        MINT_FEE_RECEIVER = address(0x0B98151bEdeE73f9Ba5F2C7b72dEa02D38Ce49Fc);
    }
```

Reply:
also we can remove the address() casting since its already an address

Reply:
Actually. `const` is better than `immutable` as it's compiled into bytecode thus has lower gas cost. 

For the suggestion of removing casting, agreed and fixed!